src/libraries/System.Net.Http.Json/tests/FunctionalTests/TestClasses.cs

```diff
@@ -36,12 +36,7 @@ public string Serialize(JsonSerializerOptions options = null)
 
     internal static class JsonOptions
     {
-        public static readonly JsonSerializerOptions DefaultSerializerOptions
-            = new JsonSerializerOptions
-            {
-                PropertyNameCaseInsensitive = true,
-                PropertyNamingPolicy = JsonNamingPolicy.CamelCase
-            };
+        public static readonly JsonSerializerOptions DefaultSerializerOptions = new JsonSerializerOptions(JsonSerializerDefaults.Web);
```

Comment:
Can we also add an assert for the new `JsonNumberHandling` setting in this method?
https://github.com/dotnet/runtime/blob/92e06eb003385438c75362c39b14214fae3fdc05/src/libraries/System.Net.Http.Json/tests/FunctionalTests/TestClasses.cs#L63-L67